docs/sql-ref-datatypes.md

```diff
@@ -119,10 +119,10 @@ from pyspark.sql.types import *
 
 |Data type|Value type in Python|API to access or create a data type|
```

Comment:
Let's also add a comment that we should fix `python/docs/source/user_guide/sql/type_conversions.rst`. You could use `<!-- comment -->`